_posts/2018-06-25-refactoring-with-design-patterns.markdown

```diff
@@ -192,3 +190,6 @@ Taxes.new.net_salary(1000, "BRA")
 ```
 
 Now if we need to add a new country we just need to create a new strategy and add to the context class. With this refactor we can keep all the concerns separated and the code is cleaner and easier to maintain.
+
+<h2 id="factory-method">Next steps</h2>
```

Comment:
@cleicar you forgot to change the `id` on the `h2` 😛

Reply:
hahaha I blame the copy and paste